New README.md

rebeccahum opened the pull request:
As per #261, we have a `readme.txt` which is not aesthetically pleasing.  I have used the contents from that file and created a `README.md` with Markdown formatting.

trepmal commented:
We won't want to remove the readme.txt entirely, as that's still needed for the plugin hosted on WordPress.org. https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/ (there's a validator there in case things need to be tweaked/updated)

trepmal commented on README.md:
```diff
@@ -17,68 +18,68 @@ On the frontend, use the [Co-Authors Plus template tags](http://vip.wordpress.co
+> If you've just installed Co-Authors Plus, you might notice that the bylines are being added in the backend but aren't appearing on the frontend. You'll need to [add the template tags to your theme](http://vip.wordpress.com/documentation/incorporate-co-authors-plus-template-tags-into-your-theme/) before the bylines will appear.
 
-= What happens to posts and pages when I delete a user assigned to a post or page as a coauthor? =
+* What happens to posts and pages when I delete a user assigned to a post or page as a coauthor? =
```
stray `=` at the end

trepmal commented on README.md:
```diff
@@ -17,68 +18,68 @@ On the frontend, use the [Co-Authors Plus template tags](http://vip.wordpress.co
+* How do I add Co-Authors Plus support to my theme?
 
-If you've just installed Co-Authors Plus, you might notice that the bylines are being added in the backend but aren't appearing on the frontend. You'll need to [add the template tags to your theme](http://vip.wordpress.com/documentation/incorporate-co-authors-plus-template-tags-into-your-theme/) before the bylines will appear.
+> If you've just installed Co-Authors Plus, you might notice that the bylines are being added in the backend but aren't appearing on the frontend. You'll need to [add the template tags to your theme](http://vip.wordpress.com/documentation/incorporate-co-authors-plus-template-tags-into-your-theme/) before the bylines will appear.
```
I'm not totally sold on the `>`/quote indenting here. I think a regular paragraph would be fine

trepmal commented on README.md:
```diff
@@ -17,68 +18,68 @@ On the frontend, use the [Co-Authors Plus template tags](http://vip.wordpress.co
+> When a user is deleted from WordPress, they will be removed from all posts for which they are co-authors. If you chose to reassign their posts to another user, that user will be set as the coauthor instead.
 
-= Can I use Co-Authors Plus with WordPress multisite? =
+* Can I use Co-Authors Plus with WordPress multisite? =
```
another stray `=`

trepmal commented on README.md:
```diff
@@ -17,68 +18,68 @@ On the frontend, use the [Co-Authors Plus template tags](http://vip.wordpress.co
+> Yep! There's a template tag called `coauthors_wp_list_authors()` that accepts many of the same arguments as `wp_list_authors()`. Look in template-tags.php for more details.
 
-== Upgrade Notice ==
+## Upgrade Notice
```
I the context of the md file for Github, we can lose the "upgrade notice" section. That's specific to how WordPress displays pending updates within WordPress.

rebeccahum commented:
@trepmal Added the readme.txt back, removed stray =s, removed quotes and removed "Update notice" section.  Let me know if there is anything else that needs to be changed, thanks!
